packages/ui/src/components/modals/ConditionModal.tsx

```diff
@@ -115,8 +174,42 @@ const Component: React.FC<Props> = (props) => {
         setSelectedOperatorOption(option)
     }
 
-    // Value
-    const [showNumberValue, setShowNumberValue] = React.useState(true)
+    // Comparison Type Dropdown
+
+    const setOperatorOptions = (option?: ComparisonTypeOption) => {
+        if (option === stringComparisonTypeOption) {
+            operatorOptions = stringEqualOperatorOptions
```

Comment:
I think since you changed it to use `allOperatorOptions` instead of `operatorOptions` you can remove this

Reply:
No, we still need this; the set of available operator options depends on the comparison type.